library/src/lib/calendar/calendar-views/calendar2-month-view/calendar2-month-view.component.html

```diff
@@ -1,3 +1,22 @@
-<p>
-  calendar2-month-view works!
-</p>
+<div class="fd-calendar__months">
+  <ul class="fd-calendar__list">
+      <li
+          class="fd-calendar__item"
+          *ngFor="let month of monthNames; let i = index"
+          [ngClass]="{
+              ' fd-calendar__item--current': i + monthOffset === currentMonth,
+              ' is-selected': i + monthOffset === monthSelected
+          }"
+          (click)="selectMonth(i, $event)"
+      >
```

Comment:
can we not put these on their own line? I know VSCode does it by itself but it's so ugly

Reply:
done